package.json

```diff
@@ -35,7 +35,7 @@
     "typescript": "3.6.3"
   },
   "engines": {
-    "node": "10.16.3"
+    "node": "10"
```

Comment:
Hmm other instances use `^10` instead of just `10`

Reply:
Thanks